hicommonwealth/commonwealth
# Remove deprecated `render` method

## Link to Issue
Closes: #3836

## Description of Changes
This PR removes `DEPRECATED_ReactRender.ts` file with `render` method that is legacy from mithril interop layer.
- `render` is replaced with regular jsx in several places in the app
- removed `link` method and now it is replaced with either `ExternalLink` and `Link` components
- removed `getAvatar` method from profile controller and chain info controller - controllers should not be responsible for rendering components - now `Avatar` components handles those cases
-`render_quill_delta.ts` file is all green because file extension was changed from `ts` to `tsx` - the main changes in the file was replacing `render` with `jsx`

## Test Plan
- App should work the same way as earlier
- Click around the app and check if user avatars render correctly
- Click around the app and check if quill renders content properly (eg in threads, comments, search results)
